wpeverest/everest-forms
# Enhancement - New add-ons are listed on the add-ons page.

### All Submissions:






### Changes proposed in this Pull Request:



New add-ons from GetResponse and Sendinblue are listed on the add-ons page.

### How to test the changes in this Pull Request:

Verify, whether new add-ons from GetResponse and Sendinblue are listed or not on the add-ons page. And also verify, whether the installation/activation/deactivation of these add-ons works appropriately.
### Types of changes:




### Other information:




### Changelog entry

> Enhancement - New add-ons are listed on the add-ons page.
